clients/src/test/java/org/apache/kafka/common/logging/Log4jControllerTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.logging;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+import java.util.Hashtable;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class Log4jControllerTest {
+
+    @Test
+    public void testGetLoggers() {
+        Hashtable<String, Logger> x = new Hashtable<>();
+        Logger h1 = new Logger("h1"){};
+        h1.setLevel(Level.INFO);
+        Logger h2 = new Logger("h2"){};
+        h2.setLevel(Level.WARN);
+        x.put("h2", h1);
```

Comment:
nit: Could we use more than one character for variable names? Test code typically gets read and edited as much as production code, so it would be good to hold it to the same quality standard